.gitignore

```diff
@@ -1,3 +1,4 @@
 /restic
 /.vagrant
 /doc/_build
+/.vscode
```

Comment:
This does not belong here. If you want to ignore that dir in your checkout of the repo, add the directory to `.git/info/exclude`. We don't collect ignores for various editors here :)

Reply:
I honestly think it is an exaggeration to be picky with `.gitignore`, but, whatever. It is your project.